package.json

```diff
@@ -8,18 +8,21 @@
 	"main": "dist/pacman-contribution-graph.min.js",
 	"scripts": {
 		"build": "webpack --config webpack.prod.js",
-		"dev": "webpack --progress --config webpack.dev.js",
+		"start": "node dist/index.js --username=diogocarrola --output=svg > assets/pacman-contributions.svg",
```

Comment:
I tried the start command, and it's not working :( . Also there's no index.js file inside dist folder